### DISCUSSION TITLE
cpresources on a elastic container / load balanced enviroment

### DISCUSSION THREAD

**Discussion by pnpaes** [craftcms/cms#13090] [General]
Hi all,

How to mitigate the of "404" control panel assets dynamically generated into cpresources in a load balanced multi in stance environment? I do not want to use sticky sessions because that is not ideal in my current setup.

Any suggestion is highly appreciated.

@brandonkelly @angrybrad do you have a standard solution for this? I guess this is a common issue/request.

**Comment by angrybrad** [comment]
Assuming you're on a recent Craft version, cpresources are generated on demand, so it shouldn’t matter if they don't have a common shared storage location (although you can certainly do that, too).

See: https://craftcms.com/knowledge-base/configuring-load-balanced-environments#cpresources-folder

Lots of other good stuff in that article to consider, as well.
